chromium/preloadlist/preloadlist.go

```diff
@@ -47,6 +47,33 @@ type PreloadList struct {
 	Entries []Entry `json:"entries"`
 }
 
+// PolicyType represents the policy under which the domain is a part of the preload list
```

Comment:
nit: I think ".. under which the domain was added to the preload..." would read better.

Reply:
Done.